### PR TITLE
 #479 - More updates to match checkstyle rules - Added comments to

### PR DESCRIPTION
**Pull request by mikewalch**
@param; Limited all lines 100 characters; Turned on ability to add
comments that turn off formatter

This gets the number of warnings down to 198

```
$ find -type f -name checkstyle-results-java.xml | xargs grep warning | wc -l
198
```

### CODE REVIEW

**Comment by mikewalch** [comment]
Warnings are now down to 159

File: modules/api/src/main/java/io/fluo/api/types/TypeLayer.java [hunk 1]
```diff
@@ -198,17 +199,19 @@
  * 
  * <pre>
  * {@code
- *   //pretend this method has curly braces, getting javadoc to work w/ less than and curly braces is too hard
+ *   // pretend this method has curly braces, getting javadoc to work w/ less than and
```
Comment (keith-turner) [inline modules/api/src/main/java/io/fluo/api/types/TypeLayer.java]: Did you look at how this javadoc renders?  If not I can take a look .  I remeber wriring all this code in javadoc being tricky.

File: modules/core/src/main/java/io/fluo/core/impl/TransactorCache.java [hunk 2]
```diff
@@ -70,7 +70,10 @@ private void logTimedoutTransactor(Long transactorId, long lockTs, Long startTim
         lockTs);
   }
 
-  public void addTimedoutTransactor(final Long transactorId, final long lockTs, final Long startTime) {
+  // @formatter:off
```
Comment (keith-turner) [inline modules/core/src/main/java/io/fluo/core/impl/TransactorCache.java]: why turn off formatting here?

File: modules/api/src/main/java/io/fluo/api/types/TypeLayer.java (hunk 1 above)
Comment (mikewalch) [inline modules/api/src/main/java/io/fluo/api/types/TypeLayer.java]: I didn't take a look but I just did and made some fixes in 4a7ebabe4b

File: modules/core/src/main/java/io/fluo/core/impl/TransactorCache.java (hunk 2 above)
Comment (mikewalch) [inline modules/core/src/main/java/io/fluo/core/impl/TransactorCache.java]: I turned off formatting as the formatter was putting the method on one line that is > 100 chars which cause checkstyle to fail.  This happens often with string text blocks but you can just break them up.  For some reason (not sure), a few methods/statements would go back to >100 char lines if the formatter was re-run.  Therefore, I just turned off formatting at these locations.  

In 4a7ebabe4b672, I added comments as to why formatting is turned off in each location.

**Comment by keith-turner** [comment]
+1

**Pull request merged**